Doc/library/os.rst

```diff
@@ -1608,6 +1589,7 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
 
    .. versionadded:: 3.3
 
+
```

Comment:
```suggestion
```

Reply:
Well I've had looked at other places where subsequent `.. data::` are used (e.g. at `RWF_NOWAIT` or `environ`)... and there two newlines are used.

Reply:
@AA-Turner If you tell me, what you prefer (two newlines as in most other positions) or just one... I could adapt the commit or resolve this point here. :-)

Reply:
I've no real preference, the suggestion was as I think the whitespace change ought be in its own PR -- this one should soley do the move.

Reply:
Fine for me... brought the whitespace back in a new commit.